script/runc.spec

```diff
@@ -0,0 +1,64 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _sbindir /usr/local/sbin
+%global _mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
+
+Name: runc
+Version: %{_version}
+Release: %{_release}
+Summary: A tool for managing OCI containers
+
+License: ASL 2.0
+Source: %{name}-%{version}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: gcc-go
+BuildRequires: glibc
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+BuildRequires: libapparmor-devel
+BuildRequires: go-md2man
```

Comment:
@moypray quick note which would sound like a disagreement about this PR (I should've mentioned this in the beginning itself, my apologies), I feel the right place of an rpm specfile is in the distro's rpm git repos, as the rpmbuild process expects the spec file outside of the source tarball so including it within the source feels a little weird to me. That implies CentOS/RHEL/Fedora won't be using this specfile but rather what exists in their dist-git.

You're more than welcome to help us out in co-maintaining Fedora/CentOS runc packages if you're interested.


@cyphar I don't know if above relates to SUSE as well.

But nonetheless, if people still feel the need to include the spec within the source, I'm ok with it.